include/dds/core/status/Status.hpp

```diff
@@ -440,31 +441,29 @@ class DataOnReadersStatus
 template<typename STATUS>
 StatusMask get_status();
 
-typedef ::dds::core::status::detail::InconsistentTopicStatus InconsistentTopicStatus;
-
-typedef ::dds::core::status::detail::LivelinessChangedStatus LivelinessChangedStatus;
+#include <dds/core/status/detail/Status.hpp>
 
-typedef ::dds::core::status::detail::LivelinessLostStatus LivelinessLostStatus;
+using InconsistentTopicStatus = dds::core::status::detail::InconsistentTopicStatus;
 
-typedef ::dds::core::status::detail::OfferedDeadlineMissedStatus OfferedDeadlineMissedStatus;
+using LivelinessChangedStatus = dds::core::status::detail::LivelinessChangedStatus;
 
-typedef ::dds::core::status::detail::OfferedIncompatibleQosStatus OfferedIncompatibleQosStatus;
+using LivelinessLostStatus = dds::core::status::detail::LivelinessLostStatus;
 
-typedef ::dds::core::status::detail::PublicationMatchedStatus PublicationMatchedStatus;
+using OfferedDeadlineMissedStatus = dds::core::status::detail::OfferedDeadlineMissedStatus;
 
-class SampleRejectedState;
+using OfferedIncompatibleQosStatus = dds::core::status::detail::OfferedIncompatibleQosStatus;
 
-typedef ::dds::core::status::detail::SampleRejectedStatus SampleRejectedStatus;
+using PublicationMatchedStatus = dds::core::status::detail::PublicationMatchedStatus;
 
-typedef ::dds::core::status::detail::RequestedDeadlineMissedStatus RequestedDeadlineMissedStatus;
+using SampleRejectedStatus = dds::core::status::detail::SampleRejectedStatus;
 
-typedef ::dds::core::status::detail::RequestedIncompatibleQosStatus RequestedIncompatibleQosStatus;
+using RequestedDeadlineMissedStatus = dds::core::status::detail::RequestedDeadlineMissedStatus;
 
-typedef ::dds::core::status::detail::SampleLostStatus SampleLostStatus;
+using RequestedIncompatibleQosStatus = dds::core::status::detail::RequestedIncompatibleQosStatus;
 
-class StatusMask;
+using SampleLostStatus = dds::core::status::detail::SampleLostStatus;
 
-typedef ::dds::core::status::detail::SubscriptionMatchedStatus SubscriptionMatchedStatus;
+using SubscriptionMatchedStatus = dds::core::status::detail::SubscriptionMatchedStatus;
```

Comment:
Remove the templates on the above classes and then you can remove all these aliases.